playground/app-router/package.json

```diff
@@ -4,19 +4,14 @@
   "private": true,
   "scripts": {
     "dev:yalc": "npm run yalc && rm -rf .next && next dev --port 4011",
-    "yalc": "yalc add -- @clerk/nextjs @clerk/clerk-react @clerk/backend @clerk/types @clerk/shared",
     "dev": "next dev --port 4011",
     "build": "next build",
-    "build:yalc": "npm run yalc && rm -rf .next && next build",
     "start": "next start",
     "lint": "next lint"
   },
   "dependencies": {
-    "@clerk/backend": "file:.yalc/@clerk/backend",
-    "@clerk/clerk-react": "file:.yalc/@clerk/clerk-react",
-    "@clerk/nextjs": "file:.yalc/@clerk/nextjs",
-    "@clerk/shared": "file:.yalc/@clerk/shared",
-    "@clerk/types": "file:.yalc/@clerk/types",
+    "@clerk/nextjs": "file:../../packages/nextjs",
+    "@clerk/clerk-react": "file:../../packages/react",
```

Comment:
❓Are we sure about these changes? This is something that existed before and knowingly moved away from.

Reply:
I'm fairly confident this is not a problematic change for the next-based playgrounds, but it might not be the right choice across the board. Do you have any links I can check out to understand the problems we were having with this approach in the past?